integr8ly/cloud-resource-operator
# MGDAPI-3225 add the engine version to the redis and postres CR

## Overview

WIP PR for setting the engine versions in the postgres and redis CR's

## Verification

- Clone this branch
- Run `make cluster/prepare`
- Run `make run`
- Run `make cluster/seed/managed/postgres`
- Run `make cluster/seed/managed/redis`
- Check the Postgres CR status when complete
```yaml
status:
  message: >-
    rds instance rhmicloudsccsqf8xmcloudresourceoper-eiw4 is as expected, aws
    rds status is available
  phase: complete
  provider: aws-rds
  secretRef:
    name: example-postgres-sec
  strategy: aws
  version: '10.16'
```
- Check the Redis CR status when complete
```yaml
status:
  message: 'successfully created and tagged, aws elasticache status is available'
  phase: complete
  provider: aws-elasticache
  secretRef:
    name: example-redis-sec
  strategy: aws
  version: 5.0.6
```

## Checklist
